All.sln

```diff
@@ -2580,6 +2590,54 @@ Global
+		{1E2644A9-6B31-4350-8772-CEAAD6EE0B21}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{1E2644A9-6B31-4350-8772-CEAAD6EE0B21}.Release|Any CPU.Build.0 = Release|Any CPU
+		{1E2644A9-6B31-4350-8772-CEAAD6EE0B21}.Release|x64.ActiveCfg = Release|Any CPU
+		{1E2644A9-6B31-4350-8772-CEAAD6EE0B21}.Release|x64.Build.0 = Release|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Debug Mac|Any CPU.ActiveCfg = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Debug Mac|Any CPU.Build.0 = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Debug Mac|x64.ActiveCfg = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Debug Mac|x64.Build.0 = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Debug|x64.Build.0 = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Release Mac|Any CPU.ActiveCfg = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Release Mac|Any CPU.Build.0 = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Release Mac|x64.ActiveCfg = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Release Mac|x64.Build.0 = Debug|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Release|Any CPU.Build.0 = Release|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Release|x64.ActiveCfg = Release|Any CPU
+		{65A2F556-F14A-49F3-8A92-7F2E1E7ED3B5}.Release|x64.Build.0 = Release|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Debug Mac|Any CPU.ActiveCfg = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Debug Mac|Any CPU.Build.0 = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Debug Mac|x64.ActiveCfg = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Debug Mac|x64.Build.0 = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Debug|x64.Build.0 = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Release Mac|Any CPU.ActiveCfg = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Release Mac|Any CPU.Build.0 = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Release Mac|x64.ActiveCfg = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Release Mac|x64.Build.0 = Debug|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Release|Any CPU.Build.0 = Release|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Release|x64.ActiveCfg = Release|Any CPU
+		{9C1ECA1D-DE98-4FB7-92D0-FC45BB308E5D}.Release|x64.Build.0 = Release|Any CPU

```

Comment:
Cripes! I am guessing Mac needs all of this?

Reply:
This is just the consequence of adding a bunch more projects to the sln file. We'll get to cleaning up the configs in the future :)